client/src/components/MatchingApplicantsAndOppotunities.js

```diff
@@ -47,57 +47,81 @@ class MatchingApplicantsAndOppotunities extends Component {
     this.getApplicantSkills();
     this.getOpportunities();
   }
+  compare = (a, b) => {
+    const genreA = Number(a.percentage);
+    const genreB = Number(b.percentage);
+
+    let comparison = 0;
+    if (genreA < genreB) {
+      comparison = 1;
+    } else if (genreA > genreB) {
+      comparison = -1;
+    }
+    return comparison;
+  };
 
   render() {
-    console.log("applicantSkills", this.state.applicantSkills);
-    console.log("opportunitiesList", this.state.opportunitiesList);
+    const matchingOpportunities = this.state.opportunitiesList
+      .map(opportunity => {
+        const matching = opportunity.skills.filter(skill => {
+          const matchingSkills = this.state.applicantSkills.includes(skill);
+          return matchingSkills;
+        }).length;
+        const percentage = (
+          (matching / opportunity.skills.length) *
+          100
+        ).toFixed(0);
+        return { ...opportunity, percentage };
+      })
+      .sort(this.compare);
 
     return (
-      <div>
-        {this.state.opportunitiesList.map(opportunity => {
-          const matching = opportunity.skills.filter(skill => {
-            const matchingSkills = this.state.applicantSkills.includes(skill);
-            return matchingSkills;
-          }).length;
-          const percentage = (
-            (matching / opportunity.skills.length) *
-            100
-          ).toFixed(2);
-          return (
-            <Card
-              key={opportunity.opportunity_id}
-              as={Link}
-              to={
-                getLoggedInUserData() &&
-                `/opportunities/${opportunity.opportunity_id}`
-              }
-            >
-              <Card.Content>
-                <Card.Header>{opportunity.opportunity_title}</Card.Header>
-                <Card.Description>{opportunity.description}</Card.Description>
-                <CircularProgressbarWithChildren
-                  strokeWidth
-                  value={percentage}
-                  text={`${percentage}%`}
-                  styles={buildStyles({
-                    strokeLinecap: "butt"
-                  })}
+      <Grid stackable>
+        <Grid.Row columns={3}>
+          {matchingOpportunities.map(opportunity => {
+            return (
+              <Grid.Column>
+                {" "}
+                <Card
```

Comment:
Pass `fluid` property to Card to take the full width on a small screen.

Reply:
Done